packages/web-app-files/tests/unit/views/views.setup.js

```diff
@@ -35,6 +35,16 @@ localVue.use(GetTextPlugin, {
   silent: true
 })
 
+export const getNgettextMock = function () {
```

Comment:
You can make this mock globally and automatically available by adding it to the mocks in `tests/unit/config/jest.init.js` right after the `$gettext` mock in line 16.

Reply:
moved